src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -65,6 +68,8 @@ const RenderListFeed = (props) => {
     showScoreButton
   } = useFeed();
   const {handleUserName} = useWriteComment();
+  const {followUnfollowTopic} = usePostHook();
+  const route = useRoute();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new hook `usePostHook` is introduced here. Please ensure that the hook is used correctly and follows the rules of hooks, i.e., it's not being called conditionally or inside loops or nested functions.



<!-- This is an auto-generated comment by OSS CodeRabbit -->